src/main/java/mate/academy/springintro/security/AuthenticationService.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.springintro.security;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.springintro.repository.UserRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class AuthenticationService {
```

Comment:
Let’s  remove the service without the realization

Reply:
fix comment